plugins/Live/Model.php

```diff
@@ -130,57 +131,58 @@ private function executeLogVisitsQuery($sql, $bind, $segment, $dateStart, $dateE
 	/**
 	 * @param \Piwik\Tracker\Db|\Piwik\Db\AdapterInterface|\Piwik\Db $readerDb
 	 * @param Exception $e
-	 * @param $sql
-	 * @param array $bind
 	 * @param $segment
 	 * @param $dateStart
 	 * @param $dateEnd
 	 * @param $minTimestamp
 	 * @param $limit
+	 * @param $parameters
 	 *
 	 * @throws MaxExecutionTimeExceededException
 	 */
-    public function handleMaxExecutionTimeError($readerDb, $e, $sql, $bind, $segment, $dateStart, $dateEnd, $minTimestamp, $limit)
+    public static function handleMaxExecutionTimeError($readerDb, $e, $segment, $dateStart, $dateEnd, $minTimestamp, $limit, $parameters)
```

Comment:
If unit tests don't exist for this method can we add some? I'm not sure if it is easy to do however. Some sort of test covering this behavior would be good IMO

Reply:
that method itself is tested here
https://github.com/matomo-org/matomo/blob/148e049bc7665af470514a8f0a573973fc59a52b/plugins/Live/tests/Integration/ModelTest.php#L74-L126

Reply:
Wouldn't this test need to be adjusted? Not seeing it in this PR? Would expect seeing this new parameter and calling it static @sgiehl ?